src/App.tsx

```diff
@@ -5,7 +5,12 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovie] = useState<Movie[]>([]);
+  const addMovie = (movie: Movie) => {
+    if (!movies.find(film => film.imdbId === movie.imdbId)) {
```

Comment:
move this expression to variable

Reply:
fixed, I misunderstood it the first time..